src/Microsoft.AspNetCore.TestHost/project.json

```diff
@@ -34,6 +34,11 @@
       "dependencies": {
         "System.Diagnostics.Contracts": "4.3.0-*"
       }
+    },
+    "netstandard1.5": {
```

Comment:
why did TestHost need to change? Shouldn't it have been fine building against the 1.3 version of Hosting?